### ISSUE TITLE
Add endpoint to return Direct projects based on a billing account

### ISSUE THREAD

**Issue by ajefts** [topcoder-platform/tc-billing-account-service#2]

**Comment by skyhit** [comment]
@ajefts should't this more fit into the https://github.com/appirio-tech/tc-direct-project-service scope.  as you need to project info, not billing account, right?

**Comment by ajefts** [comment]
Yes, I agree.  Can we currently filter direct projects by a billing account ID filter?

**Comment by skyhit** [comment]
@ajefts this is no listing project endpoint, so no filtering too
